core/jobs/batch_jobs/exp_validation_jobs_test.py

```diff
@@ -0,0 +1,96 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.exp_validation_jobs."""
+
+from __future__ import annotations
+
+from core.constants import constants
+from core.jobs import job_test_utils
+from core.jobs.batch_jobs import exp_validation_jobs
+from core.jobs.types import job_run_result
+from core.platform import models
+
+(exp_models, ) = models.Registry.import_models(
+    [models.NAMES.exploration])
+
+
+class GetExpRightsWithDuplicateUsersJobTests(
+    job_test_utils.JobTestBase):
+
+    JOB_CLASS = exp_validation_jobs.GetExpRightsWithDuplicateUsersJob
+
+    EXPLORATION_ID_1 = '1'
+    EXPLORATION_ID_2 = '2'
+    EXPLORATION_ID_3 = '3'
+    USER_ID_1 = 'id_1'
+    USER_ID_2 = 'id_2'
+    USER_ID_3 = 'id_3'
+
+    def test_run_with_duplicate_user_rights(self) -> None:
```

Comment:
add more tests, at least: one for zero models, one for one correct and one incorrect model

Reply:
I will add a test for zero models. The test that you've commented on here has 2 correct and 1 incorrect in it. What other test would you like added?